server/proxy.js

```diff
@@ -22,7 +22,7 @@ const proxyExternalHost = (
         options.headers["Authorization"] = `Bearer ${token}`;
       }
       if (
-        host === Config.auth.ispengestopp.host ||
+        (host === Config.auth.ispengestopp.host && !parseReqBody) ||
```

Comment:
Kan vurdere om dette også skulle vært en del av auth-configen (som bearerHeader). Måtte i så fall vært en funksjon av parseReqBody/req.method. Men hvis det bare er aktuelt for disse tre er det kanskje like greit med en if her 🙂

Reply:
Tenker å la det være sånn nå, og så kan vi fjøre om ispengestopp til å forvente en vanlig "nav-personident"-token i stedet. Da er det kun smreg og flex som trenger en sånn if 👍